src/Polly.Core/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,4 @@
 ﻿#nullable enable
```

Comment:
I should update my automation PR to move things to the Shipped file too...

Reply:
https://github.com/App-vNext/Polly/pull/2062/commits/2ea7e119ed67ff45ca6e4324517c2f7291f9c701